### ISSUE TITLE
Build failure with servant 0.16

### ISSUE THREAD

**Issue by snoyberg** [ch1bo/servant-exceptions#9]
```
Building library for servant-exceptions-0.1.1..
[1 of 1] Compiling Servant.Exception ( src/Servant/Exception.hs, dist/build/Servant/Exception.o )

src/Servant/Exception.hs:42:52: error:
    Module
    ‘Servant.Server.Internal.RoutingApplication’
    does not export
    ‘Delayed(..)’
   |
42 | import Servant.Server.Internal.RoutingApplication (Delayed (..))
   |               
```

**Comment by dbaynard** [comment]
It looks like servant 0.16 has [many breaking changes](https://hackage.haskell.org/package/servant-0.16/changelog) which affect this library (e.g. renaming error types).

**Comment by ch1bo** [comment]
Should be solved by #10
